husky_components/ckeditor/main.js

```diff
@@ -170,15 +170,19 @@ define([], function() {
         },
 
         renderStartTemplate: function() {
-            var $content = $(this.$el.val()),
-                text = $content.text(),
-                $trigger = $(
-                    [
-                        '<textarea class="form-element ckeditor-preview" placeholder="', this.options.placeholder, '">',
-                        text,
-                        '</textarea>'
-                    ].join('')
-                );
+            var text, $trigger;
+            try {
+                text = $(this.$el.val()).text();
```

Comment:
I would do something like `$('<div>' + this.$el.val() + '</div>').text()` because the catch will not work correctly if there is example a tag inside the text.

Reply:
I didn't check that case, only theorized about it. I guess using `<body>` instead of `<div>` would be even better. I'll test that.